testsuite/features/reposync/srv_disable_scheduled_reposync.feature

```diff
@@ -1,13 +1,23 @@
-# Copyright (c) 2019-2023 SUSE LLC
+# Copyright (c) 2019-2024 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-Feature: Delete the scheduled task for mgr-sync-refresh
+Feature: Do not let Taskomatic tasks interfer with our tests
 
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
-  Scenario: Delete scheduled mgr-sync-refresh task
+  Scenario: Disable scheduled reposyncs
     When I follow the left menu "Admin > Task Schedules"
     And I follow "mgr-sync-refresh-default"
     And I choose "disabled"
     And I click on "Update Schedule"
+    # Workaround https://bugzilla.suse.com/show_bug.cgi?id=1225740
+    And I click on "Delete Schedule"
```

Comment:
I would not delete it, but just let it failing until they fix it.

Reply:
Then we have random race conditions with reposyncs...